drivers/pure/flasharray/realmsTypes.go

```diff
@@ -0,0 +1,55 @@
+package flasharray
```

Comment:
Same here, we won't need most of these fields I don't think. Mainly just the Name, Destroyed, and possibly QoS